Description: Fix build failure with GCC 4.7
 - src/tinyxpath/node_set.cpp: Include "../main.h" for HAVE_STDINT_H.
Bug-Debian: http://bugs.debian.org/667310
Author: Matej Vela <vela@debian.org>
Last-Update: 2012-05-18

--- opencity-0.0.6.4stable~/src/tinyxpath/node_set.cpp
+++ opencity-0.0.6.4stable/src/tinyxpath/node_set.cpp
@@ -22,6 +22,7 @@
 distribution.
 */
 
+#include "../main.h"
 #include "node_set.h"
 #ifdef HAVE_STDINT_H
 	#include "stdint.h"			// for intptr_t (32/64 bits void* pointer compatibility)
